target/linux/ipq40xx/mikrotik/base-files/etc/board.d/01_leds

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
On ipq40xx, we have base-files in target (not subtarget) for now. Please add the devices there.

If you intend to split this (I personally don't see a clear preference here), please do a separate "ipq40xx: split base-files across subtargets" PR for this subject.

Reply:
Then I do not intend to split the base files.